### DISCUSSION TITLE
[consult-ripgrep] - Wrong type argument: overlayp, nil

### DISCUSSION THREAD

**Discussion by puang59** [minad/consult#1068] [Q&A]
I am using doom emacs and when I do M-x consult-grep or ripgrep and press enter, I get the error "Wrong type argument: overlayp, nil"
Im not quite sure how to use this tool or if there is something wrong with my config.

<img width="349" alt="Screenshot 2024-08-17 at 7 06 24 PM" src="https://github.com/user-attachments/assets/cb30ca70-0bb8-411d-b694-1269c02efc73">

**Comment by minad** [comment]
See https://github.com/minad/consult?tab=readme-ov-file#bug-reports.
